### PR TITLE
KAFKA-6057: Users forget `--execute` in the offset reset tool

### PR DESCRIPTION
**Pull request by gilles-degols**
Add a small warning note when the user does not use the --execute flag.

### CODE REVIEW

**Comment by gilles-degols** [comment]
retest this please

**Comment by gilles-degols** [comment]
retest this please

File: core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala
Comment (hachikuji) [inline core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala]: Would it make sense to print this on stderr instead? I was thinking that might be the command more unix-friendly (e.g. the offsets can be piped to another command more easily).

**Review by hachikuji — COMMENTED** [review]
Thanks for the patch and sorry for getting to it so late. Just had a minor suggestion for consideration.

File: core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala
Comment (gilles-degols) [reply core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala]: Indeed, it would be better on stderr, I will update it.

**Review by hachikuji — APPROVED** [review]
LGTM. I will push one formatting nit (space after the `if`) before merging.

**Comment by hachikuji** [comment]
@gilles-degols Apologies for the delay. I was about to merge this when I found that trunk had changed to remove the --execute option entirely. I think this was a mistake since we should have at least allowed for a deprecation window before removing the option, so I had to scramble a little bit to fix it for 1.1. Now that that is fixed, I went ahead and rebased and updated this patch. I will go ahead and merge if the changes look good to you.

**Comment by hachikuji** [comment]
Thanks for the PR. I'm going to go ahead and merge.

**Pull request merged**
